katas/content/single_qubit_gates/index.md

```diff
@@ -365,6 +365,76 @@ All the basic gates we will be covering in this kata are part of the <a href="ht
     "solutionPath": "./sign_flip_on_zero/solution.md"
 })
 
+@[section]({
+    "id": "identity_gate",
+    "title": "Identity Gate"
+})
+
+The identity gate is mostly here for completeness, at least for now. It will come in handy when dealing with multi-qubit systems and multi-qubit gates. It is represented by the identity matrix, and does not affect the state of the qubit.
+  
+<table style=\"border:1px solid\">
```

Comment:
```suggestion
<table>
  <tr>
    <th>Gate</th>
    <th>Matrix</th>
    <th>Ket-Bra</th>
    <th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
    <th>Q# Documentation</th>
  </tr>
  <tr>
    <td>$I$</td>
    <td>$\begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}$</td>
    <td>$|0\rangle\langle0| + |1\rangle\langle1|$</td>
    <td>$I|\psi\rangle = |\psi\rangle$</td>
    <td><a href=\"https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.i\">I</a></td>
  </tr>
</table>
```

Reply:
Since the style comes from the consumer of the npm package, there's no need to include style in the tables.